examples/tutorials/research/02_advanced_usage_of_research.ipynb

```diff
@@ -57,7 +57,7 @@
    "source": [
```

Comment:
<p><code>os.environ["CUDA_VISIBLE_DEVICES"] = "1, 2"</code>  is unnecessary now</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/495/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>